docs/src/data/fetch-stars.ts

```diff
@@ -0,0 +1,17 @@
+export async function fetchStars() {
```

Comment:
Not a place to store business logic. Please go through Astro best practices and place the file at the apt place.
Also, the name of the file should be `github.ts` which will hold all GitHub utilities.